Sources/Hyperspace/Service/Backend/BackendServiceProtocol.swift

```diff
@@ -87,6 +87,6 @@ extension BackendServiceError: Equatable {
 
 extension AnyError: BackendServiceErrorInitializable {
     public init(_ backendServiceError: BackendServiceError) {
-        self.init(backendServiceError)
+        self.init( backendServiceError as Error)
```

Comment:
Nit picking here - but can we remove the leading space?

Reply:
Done